Part2

```diff
@@ -0,0 +1,17 @@
+Matthew Danielson
+7/5/16
+Part 2; Questions
+
+1) 
+int fxn(int, float, int);
+int main(){
+	int a = 1;
+	float b = .1;
+	int c = 1;
+	fxn(int a, float b, int c);
+}
```

Comment:
oop, actually, i just realized you didn't include your variable names in the method declaration on line 6!